modules/rostests/apitests/CMakeLists.txt

```diff
@@ -51,6 +51,7 @@ add_subdirectory(spoolss)
 add_subdirectory(psapi)
 add_subdirectory(umkm)
 add_subdirectory(user32)
+add_subdirectory(user32/makeimg)
```

Comment:
This should be in the user32's CMakeLists.

Reply:
Moved. Thanks.